sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2285,6 +2292,19 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setNumFileShards(numFileShards).build();
     }
 
+    /**
+     * Control how many parallel streams are used when using Storage API writes. Applicable only
+     * when also setting {@link #withTriggeringFrequency}. To let runner determine the sharding at
+     * runtime, set {@link #withAutoSharding()} instead.
```

Comment:
Drive-by comment: Would this be misleading? Since auto-sharding is not yet integrated.